sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveCommandSuite.scala

```diff
@@ -122,4 +122,123 @@ class HiveCommandSuite extends QueryTest with SQLTestUtils with TestHiveSingleto
       checkAnswer(sql("SHOW TBLPROPERTIES parquet_temp"), Nil)
     }
   }
+
+  test("LOAD DATA") {
+    withTable("non_part_table", "part_table") {
+      sql(
+        """
+          |CREATE TABLE non_part_table (time TIMESTAMP, id INT)
+          |ROW FORMAT DELIMITED
+          |FIELDS TERMINATED BY ','
+          |LINES TERMINATED BY '\n'
+        """.stripMargin)
+
+      val testData = hiveContext.getHiveFile("data/files/issue-4077-data.txt").getCanonicalPath
```

Comment:
yea it's not a big deal. I also checked the `HiveTableScanSuite`, this file is really used for that issue. So I suggest we use another file like `employee.dat` and add comments to explain the file schema, which is better for review.